### PR TITLE
Change Giphy and Gravatar URL from HTTP to HTTPS

### PR DESCRIPTION
**Pull request by jonakoudijs**

### CODE REVIEW

File: _includes/person.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="row">
     <div class="col-xs-4">
       {% if include.person.gravatar %}
-        <img class="img-rounded" src="http://www.gravatar.com/avatar/{{include.person.gravatar}}?s=320" />
+        <img class="img-rounded" src="https://www.gravatar.com/avatar/{{include.person.gravatar}}?s=320" />
```
Comment (tader) [inline _includes/person.html]: just remove http(s) entirely, make it `src="://www.g...`

File: index.md [hunk 2]
```diff
@@ -12,6 +12,6 @@ form of publications and open-source projects.
 </div>
 
 <div markdown="1" class="col-md-offset-0 col-md-4 col-sm-offset-0 col-sm-4 col-xs-offset-2 col-xs-8">
-![epic](http://i.giphy.com/b09xElu8in7Lq.gif){:.image-max}
+![epic](https://i.giphy.com/b09xElu8in7Lq.gif){:.image-max}
```
Comment (tader) [inline index.md]: just remove http(s) entirely, make it `[epic](://i.g...`

**Comment by tader** [comment]
Why? We don't have HTTPS on the site itself...

**Comment by svdgraaf** [comment]
@tader I'm in favour of this change, but I agree on the `://` usage.

**Comment by jonakoudijs** [comment]
Apparently not at the moment. The reason why I made this pull request because I visited the site on https and got errors that some resources were referenced on http. And now I see that the LE certificate has been expired.

Changing it to `://` is fine for me. Will do that in a sec.

**Comment by jonakoudijs** [comment]
I changed `https:` to `:`.

**Comment by svdgraaf** [comment]
Not sure why this is still not merged, but I'll go ahead and merge it ;)

**Pull request merged**